packages/core/src/server/cms/index.ts

```diff
@@ -32,6 +32,27 @@ export type PageContentType = ContentData & {
       canonical?: string
       name?: string
       publisherId?: string
+      organization: {
+        id?: string
+        url?: string
+        sameAs?: string[]
+        logo?: string
+        image?: {
+          url: string
+          caption: string
+          id: string
+        }
+        name: string
```

Comment:
question: was it intentional to not put those as optional?

Reply:
yes! But I've made a small adjustment. It doesn’t make sense to include the organization structure data type without meaningful data.

<img width="546" alt="image" src="https://github.com/user-attachments/assets/c71c12c3-d64d-412c-948f-e03d11e30225" />

Although Google’s guidelines don’t specify required fields, I set url and logo as required. I also added a conditional check to prevent errors when a required field is missing (since the CMS returns undefined). I applied the same logic to the Address type to ensure it’s only included when it contains valid information.